pkg/volume/util.go

```diff
@@ -410,3 +410,27 @@ func JoinMountOptions(userOptions []string, systemOptions []string) []string {
 	}
 	return allMountOptions.UnsortedList()
 }
+
+// ZonesToSet converts a string containing a comma separated list of zones to set
+func ZonesToSet(zonesString string) (sets.String, error) {
+	zonesSlice := strings.Split(zonesString, ",")
+	zonesSet := make(sets.String)
+	for _, zone := range zonesSlice {
+		trimmedZone := strings.TrimSpace(zone)
+		if trimmedZone == "" {
+			return make(sets.String), fmt.Errorf("comma separated list of zones (%q) must not contain an empty zone", zonesString)
+		}
+		zonesSet.Insert(trimmedZone)
+	}
+	return zonesSet, nil
+}
+
+// ValidateZone returns:
+// - an error in case zone is an empty string or contains only any combination of spaces and tab characters
+// - nil otherwise
+func ValidateZone(zone string) error {
```

Comment:
IMHO, it's a matter of subjective preference.
I prefer to express my [intention](https://en.wikipedia.org/wiki/Intentional_programming) at call-sites rather than to in-line the implementation at call-sites even though the current implementation of the validation is trivial.
In addition, the function `ValidateZone` is not called at only 1 place.
That's why I prefer to keep it as it is.